packages/v4/patternfly-docs/content/design-guidelines/components/skeleton/skeleton.md

```diff
@@ -5,14 +5,22 @@ section: components
 
 ## Usage 
 Skeleton loading should be progressive –– it should load static text first, followed by variable elements of the UI.
+A skeleton is a type of loading state that allows you to expose content incrementally, once the structure of the page has been loaded in. A skeleton should match the exact structure of the element you’re loading in and can be used in components on top of the page, such as in a drop down menu, or a card.
 
 ### When to use a skeleton vs. a loading spinner
+- The structure of a page has loaded in but the content has not
```

Comment:
would add a sentence under the header that says:
**Use a skeleton when:**
- the structure of a page has loaded... etc